pkgs/tools/security/keysmith/default.nix

```diff
@@ -30,8 +30,8 @@ mkDerivation rec {
   postInstall = ''
     mv $out/bin/org.kde.keysmith $out/bin/.org.kde.keysmith-wrapped
     makeWrapper $out/bin/.org.kde.keysmith-wrapped $out/bin/org.kde.keysmith \
-      --set QML2_IMPORT_PATH "${lib.getLib kirigami2}/lib/qt-5.12.7/qml:${lib.getBin qtquickcontrols2}/lib/qt-5.12.7/qml:${lib.getBin qtdeclarative}/lib/qt-5.12.7/qml:${qtgraphicaleffects}/lib/qt-5.12.7/qml" \
-      --set QT_PLUGIN_PATH "${lib.getBin qtbase}/lib/qt-5.12.7/plugins"
+      --set QML2_IMPORT_PATH "${lib.getLib kirigami2}/${qtbase.qtQmlPrefix}:${lib.getBin qtquickcontrols2}/${qtbase.qtQmlPrefix}:${lib.getBin qtdeclarative}/${qtbase.qtQmlPrefix}:${qtgraphicaleffects}/${qtbase.qtQmlPrefix}" \
+      --set QT_PLUGIN_PATH "${lib.getBin qtbase}/${qtbase.qtPluginPrefix}"
```

Comment:
this shouldn't be needed anymore, because qt's `mkDerivaiton` automatically adds `wrapQtAppsHook` which will make a wrapper with these entries (as long as they're in buildInputs`)

Reply:
I did wonder why this was the only package doing this (and why I couldn't actually get the package to fail). Do we just remove the whole `postInstall` section then?

Reply:
Some people couldn't run it on WM setup. This didn't fixed it but I kept it in case of.

Reply:
@Thra11 It's likely an artifact of older packaging, because the wrappers for qt are **relatively** new.